crates/outbound-mqtt/src/lib.rs

```diff
@@ -101,11 +105,11 @@ impl v2::HostConnection for OutboundMqtt {
     }
 }
 
-fn convert_to_mqtt_qos_value(qos: Qos) -> i32 {
+fn convert_to_mqtt_qos_value(qos: Qos) -> rumqttc::QoS {
```

Comment:
Probably we can rename this function now that we use rumqttc which takes it's own enum than standard MQTT values i.e. 0,1,2

Reply:
@suneetnangia Good point, though I'm not sure any new name would end up looking very different... _grin_

Given how long builds take, and since I really really want to see if this passes release build, would it be okay if I merged anyway?  We can come back to it in a tidying PR for sure.

Reply:
Sounds good.